tests/routes/master/workers.test.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var mockery = require('mockery');
+var rewire = require('rewire');
+var path = require('path');
+var assert = require('chai').assert;
+
+var __ROOTDIR = path.join(__dirname, '..', '..', '..');
```

Comment:
would you like to use a lib for root-path? just a suggestion.
https://www.npmjs.com/package/rootpath
https://www.npmjs.com/package/app-root-dir